### PR TITLE
linking to RDMk to reflect current mapping file, incorporating change…

### PR DESCRIPTION
**Pull request by proccaserra**
…s from PR and openrefine recipe

### CODE REVIEW

**Review by daniwelter — APPROVED** [review]
Provided everything renders fine, it all looks ok to me.

File: content/recipes/infrastructure/data-catalog.md [hunk 1]
```diff
@@ -30,6 +30,15 @@ We will cover the following points:
 6. facet oriented searching
 7. minting of stable, persistent and resolvable identifiers
 
+```{tabbed} Table of Data Standards
```
Comment (FuqiX) [inline content/recipes/infrastructure/data-catalog.md:33]: I can't access the branch, here is the updated table with links.

```{tabbed} Table of Data Standards
| Data Formats  | Terminologies | Models  |
| :------------- | :------------- | :------------- |
| [JSON](http://edamontology.org/format_3464)  | <!-- TODO add a link to corresponding document -->  | <!-- TODO add a link to corresponding document -->  |
| [RDF](http://edamontology.org/format_2376)  | [DCAT v1](https://www.w3.org/TR/vocab-dcat-1/)  | [DATS](http://frog.oerc.ox.ac.uk:8080/SDSVoc/rash/DATS.html) |
| [RDF](http://edamontology.org/format_2376)  | [DCAT v2](https://www.w3.org/TR/vocab-dcat-2/) | [DATS](http://frog.oerc.ox.ac.uk:8080/SDSVoc/rash/DATS.html) |
| [JSON-LD](http://edamontology.org/format_3749) | [Schema.org](https://schema.org/)  | <!-- TODO add a link to corresponding document -->  |
```

File: content/recipes/accessibility/sftp.md [hunk 2]
```diff
@@ -254,17 +254,17 @@ For example [Rclone](https://rclone.org) is one such tool, allowing interaction
...
+> * [Example of customization for a specific institution: University of Luxembourg File Transfer](https://hpc.uni.lu/users/docs/filetransfer.html)
 >
-> [Setting up an SFTP server on CentOS](https://www.howtoforge.com/tutorial/how-to-setup-an-sftp-server-on-centos/)
+> * [Setting up an SFTP server on CentOS](https://www.howtoforge.com/tutorial/how-to-setup-an-sftp-server-on-centos/)
 >
-> {ref}`fcb-access-aspera`
+> * {ref}`fcb-access-aspera`
 >
-> [Rclone](https://rclone.org)
+> * [Rclone](https://rclone.org)
```
Comment (FuqiX) [inline content/recipes/accessibility/sftp.md:267]: [Rclone for managing files on cloud storage]((https://rclone.org)

Adding a short explanation about Rclone

**Review by FuqiX — APPROVED** [review]
It looks good to me. I also suggested some small changes.

**Pull request merged**